lms/djangoapps/certificates/views/webview.py

```diff
@@ -98,7 +98,7 @@ def _update_certificate_context(context, user_certificate, platform_name):
 
     # Translators:  The format of the date includes the full name of the month
     context['certificate_date_issued'] = _('{month} {day}, {year}').format(
-        month=user_certificate.modified_date.strftime("%B"),
+        month=strftime_localized(user_certificate.modified_date, "%B"),
```

Comment:
@pepeportela It looks like this function was never imported. Can you fix and open another PR? Thanks.